### PR TITLE
Feature/book model

### PR DESCRIPTION
**Pull request by f-medina**
## Summary
- Completed the book Model
## Trello Card
https://trello.com/c/L7vbnY7H/14-crear-modelo-de-book

### CODE REVIEW

File: src/main/java/wolox/training/models/Book.java [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    private String author;
+    private String image;
+    private String title;
+    private String subtitle;
+    private String publisher;
+    private String year;
+    private Integer pages;
+    private String isbn;
+
+
```
Comment (AleFalcon) [inline src/main/java/wolox/training/models/Book.java:17]: You must create empty constructor. This constructor's used by JPA by default

**Review by AleFalcon — CHANGES_REQUESTED** [review]

**Review by torsello — CHANGES_REQUESTED** [review]
I'm with Ale, you must create an empty constructor too.

Fran, are you using a .gitignore file? if not, please add one for intellij + java + gradle.

PD: remember, once you merge the first PR, you need to rebase this branch, so we only see the current changes

File: src/main/java/wolox/training/models/Book.java [hunk 2]
```diff
@@ -11,6 +11,9 @@
     private Integer pages;
     private String isbn;
 
+    public Book() {
```
Comment (torsello) [inline src/main/java/wolox/training/models/Book.java:14]: Fran, after pushing your fixes, please click on request review again :)

**Review by torsello — APPROVED** [review]
LGTM!

**Review by AleFalcon — APPROVED** [review]

**Pull request merged**